core/deployment/src/main/java/io/quarkus/deployment/JBossProcessor.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.deployment;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.nativeimage.RuntimeInitializedClassBuildItem;
+
+public class JBossProcessor {
```

Comment:
Minor nitpick but could we call this class `JBossThreadsProcessor` instead?

Reply:
That was the initial name I used as well, but then I changed it so that in the future if anyone wants to add anything JBoss related they wouldn't create a new class for it as we don't seem to do much for jboss dependencies... I am OK with either name.

Reply:
+1 for `JBossThreadsProcessor`. Having a separate `Processor` class for each JBoss project is a better approach IMHO